app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;}
+a:visited {color:#000;}
+a:hover   {color:#000;}
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
+  font-family: sans-serif;
+}
+
+.login {
+  background-color: white;
+  border-color: #E0E0E0;
```

Comment:
Color literals like `#E0E0E0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E0E0E0` should be written as `#e0e0e0`